addons/coupon/report/coupon_report_templates.xml

```diff
@@ -73,7 +73,7 @@
                                 <br/>
                                 <div class="mt32">
                                     <div class="text-center">
-                                        <img alt="Logo" t-att-src="'/logo?company=%d' % (o.program_id.company_id)" t-att-alt="'%s' % (o.program_id.company_id.name)" style="border:0px solid transparent; height: 50; width: 200px;" height="50"/>
+                                        <img alt="Logo" t-att-src="'/logo?company=%d' % (o.program_id.company_id)" t-att-alt="'%s' % (o.program_id.company_id.name)" style="border:0px solid transparent; height: 50px;" height="50"/>
```

Comment:
Hm, couldn't you completely remove the height in the att-alt then, seeing as it's the next attribute?

Reply:
Hm, perhaps, I'll checkl this out.